tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p><code>unitary_simulator</code> , <code>qasm_simulator</code> ,  <code>statevector_simulator</code> within double backquotes might be nice. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1344/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>